src/PAModelTests/ErrorTests.cs

```diff
@@ -55,5 +57,25 @@ public void OpenMissingSources()
             Assert.IsTrue(errors.HasErrors);
             Assert.IsNull(doc);
         }
+
+        [TestMethod]
+        public void OpenJSONMismatchMsApp()
+        {
+            (var doc, var errors) = CanvasDocument.LoadFromMsapp(PathJSONMismatchDir);
+            Assert.IsTrue(errors.HasErrors);
+            Assert.IsNull(doc);
+
+            bool containsJSONMismatch = false;
+
+            foreach (var err in errors)
+            {
+                if (err.Code == (ErrorCode)3013)
```

Comment:
nit:  can we use the enum value here instead of 3013 in the check.( Errorcode.JSONMismatch ).